### PR TITLE
feat(blobs/properties) - Display & update blob properties

### PR DESCRIPTION
**Pull request by sedouard**
Right click Select Properties:
screen shot 2015-07-28 at 5 56 16 pm

If blob is locked you can't edit it:
screen shot 2015-07-28 at 6 03 11 pm

And if it isn't, you can:

screen shot 2015-07-28 at 5 56 48 pm

screen shot 2015-07-28 at 6 35 43 pm

In the future we can add tabs on the navbar for Metadata and a PageBlob browser.

### CODE REVIEW

File: app/templates/explorer.hbs [hunk 1]
```diff
@@ -64,18 +65,18 @@
                         {{/each}}
                         {{#each blob in blobsSorted itemController="blob"}}
                             <tr {{bind-attr class="blob.selected:selected"}}>
-                                <td>
-                                    {{input type="checkbox" id=blob.model.name checked=blob.model.selected}}
-                                    <label for={{blob.model.name}}></label>
+                                <td data-properties-controller="explorer" data-properties-model="blob" data-properties-id="{{blob.model.id}}" data-properties-controller="explorer">
```
Comment (sedouard) [inline app/templates/explorer.hbs]: @felixrieseberg is there an 'ember' way to decorate app data into elements for use outside of ember? (We need this to handle a jquery event.)

File: tests/unit/controllers/blob-test.js [hunk 2]
```diff
@@ -97,6 +95,7 @@ test('it calculates lock status correctly (unlocked)', function (assert) {
         getBlobs()
         .then(blobs => {
             blobs.every(blob => {
+                console.log('HELLO!!!');
```
Comment (felixrieseberg) [inline tests/unit/controllers/blob-test.js]: Hi!

**Comment by felixrieseberg** [comment]
Wooohooo, awesome! Regarding the `data`-props: I might have an idea for how to do this without jQuery, but I'll make sure that it works and that it doesn't suck before boasting about. Want to merge now and I make another PR if I actually have something better?

**Comment by sedouard** [comment]
Sounds good.

File: tests/unit/controllers/blob-test.js (hunk 2 above)
Comment (sedouard) [inline tests/unit/controllers/blob-test.js]: lol - I gotta get in the habit of using the ember logger for debugging.

**Pull request merged**